lib/package/menu.coffee

```diff
@@ -36,11 +36,18 @@ module.exports =
         label: 'Working Directory'
         submenu: [
           {label: 'Current File\'s Folder', command: 'julia-client:work-in-current-folder'}
-          {label: 'Current Project\'s Folder', command: 'julia-client:work-in-project-folder'}
+          {label: 'Current Project\'s Folders', command: 'julia-client:work-in-project-folder'}
           {label: 'Home Folder', command: 'julia-client:work-in-home-folder'}
           {label: 'Select...', command: 'julia-client:select-working-folder'}
         ]
       }
+      {
+        label: 'Active Project',
```

Comment:
How about
```suggestion
        label: 'Environment',
```
?

Reply:
yeah sounds good.
As for this, how does `activate-environment-in-current-folder` sound to you instead of `activate-current-folder` ? IMO this sounds clearer (a bit verbose tho)

Reply:
Yeah, should be fine. It's not like anyone would need to type out those words ;)